GroupIS1024/Models/Coach.cs

```diff
@@ -1,23 +1,29 @@
-﻿namespace GroupIS1024.Models
+﻿using Newtonsoft.Json;
+
+namespace GroupIS1024.Models
 {
     public class Coach
     {
+        [JsonProperty("Coach Id")]
         public int CoachId { get; set; }
 
+        [JsonProperty("First Name")]
```

Comment:
Added JsonProperty data annotation for better display of column names in CRUD Pages to the users

Reply:
These names have spaces.  That won't be permitted in JSON, so this won't parse properly.  I'd leave this as it were.  As long as the property names match the JSON names, you should be in good shape.